olusoladavid/my-diary
# #159300877 Add home navigation to login and signup page

#### What does this PR do?

Adds a navigation link that returns the user to the homepage from both login and signup pages
#### Description of Task to be completed

- Add the existing responsive nav menu to the login and signup page
- Add a navigation link to the menu
- Point navigation link at the homepage

#### How should this be manually tested?

Review the code changes and the screenshots. Notice the "Home" link at the top right corner.

![Login page](https://user-images.githubusercontent.com/30433120/43203754-f899de6c-9016-11e8-8f67-4b05843534d0.png)


#### What are the relevant pivotal tracker stories?

[#159300877](https://www.pivotaltracker.com/story/show/159300877)
